Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8 
+
+WORKDIR /usr/src/app 
+
+COPY .synapseConfig ./
+COPY cbioportal/ ./
+COPY requirements.txt ./ 
+RUN pip install --no-cache-dir -r requirements.txt 
+
+COPY . .
```

Comment:
Be careful about your data directory.  Because you could package up data that exists in the repo

Reply:
added .dockerignore file that lists potential subfolders from all defined cohort classes